src/main/java/org/sonar/FileOutputDemo.java

```diff
@@ -0,0 +1,37 @@
+package org.sonar;
+
+import java.io.*;
+
+class FileOutputDemo 
+{   
+
+    private FileOutputDemo(){    
+    }
+    /**
+     * this is the main method
+     * @param args
+     */
+        public static void main(String[] args)
+        {              
+                FileOutputStream out; // declare a file output object
+                PrintStream p; // declare a print stream object
+
+                try
+                {
+                        // Create a new file output stream
+                        // connected to "myfile.txt"
+                        out = new FileOutputStream("myfile.txt");
+
+                        // Connect print stream to the output stream
+                        p = new PrintStream( out );
+        
+                        p.println ("This is written to a file");
+
+                        p.close();
+                }
+                catch (Exception e)
+                {
+                    throw new RuntimeException(e);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS00112)